src/reactor.cr

```diff
@@ -9,13 +9,15 @@ module Mint
   # * When --auto-format flag is passed all source files are watched and if
   #   any changes it formats the file
   class Reactor
-    @sockets = [] of HTTP::WebSocket
+    @artifacts : TypeChecker::Artifacts?
+    @auto_format : Bool
     @error : String?
-    @ast : Ast = Ast.new
-    @script = ""
     @host : String
     @port : Int32
-    @auto_format : Bool
+
+    @sockets : Array(HTTP::WebSocket) = [] of HTTP::WebSocket
+    @script : String = ""
+    @ast : Ast = Ast.new
```

Comment:
Inference to the rescue!

```suggestion
    @sockets = [] of HTTP::WebSocket
```

The removed declarations are repeated with the `getter` calls below.